frontend/public/components/overview/_project-overview.scss

```diff
@@ -24,6 +24,9 @@
     @media (min-width: $screen-md-min) {
       grid-area: alert;
     }
+    .co-icon-and-text {
```

Comment:
There is also `.co-icon-and-text` style in the `_icon-and-text.scss` file.  Shouldnt we put it there ?

Reply:
Since this is a contextual override, I prefer keeping it with the context.  That way, if the context goes away, the override goes with it.